src/data/Reserves.ts

```diff
@@ -78,7 +84,10 @@ export function useGetNearAllPool() {
   });
 }
 
-export function useNearPairs(currencies: [Currency | undefined, Currency | undefined][]): [PairState, Pair | null][] {
+export function useNearPairs(
+  currencies: [Currency | undefined, Currency | undefined][],
+  poolTypes = [PoolType.SIMPLE_POOL],
```

Comment:
actually `useNearPairs` hook should be identical to `usePairs` hook. when I say identical, I mean input arguments and output should match because we are using those hooks conditional based on chain selected. please check here how we are using it `src/data/multiChainsHooks.ts`.

Also have some doubt ? for swap routing, only these changes are required ? or do we need any sdk update ?

Reply:
gotcha, I was thinking more on this after (re: some of the above comments) and we can make this `useNearPairs` the exact equivalent pulling just the simple pools as you mention. I'll need to make some sdk updates and likely add a fee param to the Pair object to handle the 0.2% fee pools on Near